infra/resources/_modules/cdn/cdn.tf

```
@@ -26,9 +26,27 @@ resource "azurerm_cdn_endpoint" "this" {
   }
 
   delivery_rule {
-    name  = "EnforceHTTPS"
+    name  = "WellKnownRewrite"
     order = 1
 
+    request_uri_condition {
+      operator = "BeginsWith"
+      match_values = [
+        "https://wallet.io.pagopa.it/.well-known/"
+      ]
+    }
+
+    url_rewrite_action {
+      source_pattern          = "/.well-known/"
+      destination             = "/assets/"
```

What do you think? There may also be other paths to reach in the future
```suggestion
      destination             = "/well-known/"
```